pipeline/transformer/transformer_mets/src/main/scala/weco/pipeline/transformer/mets/transformer/ImageUtils.scala

```diff
@@ -27,19 +27,24 @@ object ImageUtils {
     )
 
   private final val thumbnailDim = "200"
+  private final val thumbnailPathSuffix =
+    s"full/!$thumbnailDim,$thumbnailDim/0/default.jpg"
 
-  def buildThumbnailUrl(bnumber: String,
-                        validThumbnailFile: FileReference): Option[String] =
+  // The /thumbs URL is routed to DLCS which handles only images
+  // other asset types are routed to the iiif-builder service at /thumb
+  val imagesThumbBaseUrl = "https://iiif.wellcomecollection.org/thumbs"
+  val notImagesThumbBaseUrl = "https://iiif.wellcomecollection.org/thumb"
```

Comment:
nit:

```suggestion
  val othersThumbBaseUrl = "https://iiif.wellcomecollection.org/thumb"
```

This is possibly bikeshedding over a name, but this would line up the two variables and make the distinction super-duper obvious.